framework/base/src/types/managed/wrapped/traits/fixed_token_supply.rs

```diff
@@ -0,0 +1,31 @@
+use crate::imports::{BigUint, ErrorApiImpl, ManagedTypeApi};
+
+pub trait FixedSupplyToken<M: ManagedTypeApi> {
```

Comment:
Looking great, but we should absolutely have some example in a contract and some tests.